examples/bootstrap-script/README.md

```diff
@@ -0,0 +1,116 @@
+# Bootstrapping your users
+
+Before spawning a notebook to the user, it could be useful to 
+do some preparation work in a bootstrapping process.
+
+One common use case is at follows:
+
+* You are using an LDAPAuthenticator and have no home directory for your user on the host
```

Comment:
I would like to *not* have LDAPAuthenticator be part of the example, since doing this is actually insecure and we recommend against it in the LDAPAuthenticator README: https://github.com/jupyterhub/ldapauthenticator#configuration-note-on-local-user-creation